src/projects/demos/1/README.md

```diff
@@ -1,4 +1,73 @@
 # Demo & Pitch Night 1
+
 ## 30 November 2023
 
-Coming soon...
\ No newline at end of file
+## Objectives
+
+Demos are intended to be a check-in of your progress so far and a way for your whole team to show off their work, as well as an opportunity to practice pitching your project to the public.
+
+This will also let us understand your progress and blockers in order to help you best.
+
+Since Tech Start is built off 3 pillars, development, strategy, and design, the demos will also focus on these 3 aspects.
+Feel free to divide the allocated time between them as you see fit.
+
+In addition, there may or may not be pizza or snacks present.
+
+## Time Limit: 15 minutes
+
+## Suggested talking points
+
+These are just suggestions and not at all mandatory, but may help get you started.
+
+### Strategy
+
+- What is your Total Market, Serviceable Market and Obtainable Market?
+- What is your Market Positioning?
+- What is your unique value?
+- Competition Analysis in terms of market, features etc.
+- Does the user research validate the business plan so far?
+
+### Design
+
+- After doing user research and synthesizing the data, what did you learn about your users?
+- What opportunities/ edge have you discover for your product?
+- What were some challenges you faced in this process (user research/ synthesis)?
```

Comment:
What were some challenges you faced in the user research and synthesis process and what did you try to overcome them?